roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -226,7 +226,11 @@ listeners:
     x_forwarded: true
 
     resources:
-      - names: [client]
+      - names:
```

Comment:
For this one, we should probably introduce a new variable in `roles/matrix-synapse/defaults/main.yml` like this:

```yaml
matrix_synapse_http_listener_resource_names: ['client']
```

We can use its value here like this:
```
    resources:
      - names: {{ matrix_synapse_http_listener_resource_names|to_json }}
        compress: false
```

In `group_vars/matrix_servers`, we can do the override depending on Dimension configuration. Example:

```yaml
matrix_synapse_http_listener_resource_names: |
  {{
    ['client']
    +
    (['openid'] if matrix_dimension_enabled and not matrix_synapse_federation_enabled else [])
  }}
```

Reply:
Resolved in 3f4bc9b.